src/prerender.js

```diff
@@ -21,33 +26,18 @@ module.exports = async () => {
 	}
 	const routes = Object.values(navRoutes)
 		.concat(Object.values(blogRoutes))
+		.concat(Object.values(tutorialRoutes))
 		.concat(guideRoutes)
+		.filter(route => !/\/:/.test(route.path))
 		.sort((a, b) => a.path.localeCompare(b.path));
 
 	let preactVersion;
 	try {
 		preactVersion = (await fetchRelease('preactjs/preact')).version;
 	} catch {}
 
-	const pageData = routes.flatMap(function map(route) {
+	const pageData = routes.flatMap(route => {
 		const url = route.path;
-		// Expand `/:x?` fields in URLs to prerender all URLs
-		const FIELD = /\/:([\w.-]+)([*+?]?)/i;
-		const field = FIELD.exec(url);
-		if (field) {
-			let start = url.substring(1, field.index);
-			let dir = path.resolve(__dirname, '../content/en', start);
-			const paths = fs
-				.readdirSync(dir)
-				.filter(rep => rep[0] !== '.' && rep.match(/\.md$/i))
-				.map(rep => rep.replace(/(^index)?\.md$/i, ''));
-			return paths
-				.flatMap(rep => {
-					let path = url.replace(FIELD, '/' + rep).replace(/\/$/, '');
-					return map(Object.assign({}, route, { path }));
-				})
-				.sort((a, b) => a.url.localeCompare(b.url));
-		}
```

Comment:
We already list all routes (now with this PR, anyhow) in `config.json` so expanding `/:x?` routes adds nothing.

Probably worth rethinking `config.json` in general, but that's a task for another day.